src/main/java/redis/clients/jedis/json/RedisJsonCommands.java

```diff
@@ -55,6 +55,10 @@ default String jsonSetLegacy(String key, Object pojo, JsonSetParams params) {
 
   <T> T jsonGet(String key, Class<T> clazz, Path... paths);
 
+  String jsonMerge(String key, Path2 path, Object object);
+
+  String jsonMerge(String key, Path path, Object pojo);
+
```

Comment:
Put these after all jsonSet and before all jsonGet. Make similar change in all other interfaces and classes.